job_runner/Raw_Gent/tools.py

```
@@ -3,119 +3,126 @@
+    """Resolve the cloned repo path from ADK session state or environment."""
+    if context is not None:
+        try:
+            invocation_context = getattr(context, "_invocation_context", None)
+            session = getattr(invocation_context, "session", None)
+            state = getattr(session, "state", None)
+            if isinstance(state, dict):
+                repo_path = state.get("repo_path")
+                if repo_path:
+                    return str(repo_path)
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)